dbcrossbar/src/clouds/gcloud/client.rs

```diff
@@ -131,7 +131,7 @@ impl Client {
                 let resp_result = self
                     .client
                     .get(url.as_str())
-                    .bearer_auth(token.as_str())
+                    .bearer_auth(token.token().expect("Token missing"))
```

Comment:
`dbcrossbar` tries very hard to avoid `expect`, except in cases where it's obvious that the code can never fail at runtime (except in the case of bugs). I'd prefer to raise an error even in unlikely cases, unless we can clearly document why this could never fail.

Reply:
will fix. These are literally the first lines of Rust I've written in my life, so the exact implications of  `expect` is still not entirely clear to me.

Reply:
`expect` triggers a hard shutdown of the entire Rust process via `panic!()` and (in the case of `dbcrossbar`) calling `abort()` to dump a core file. It's normally appropriate for things which should "never occur" but have somehow happened anyway. In throwaway code, you might also use it for error handling. And it's fine in test suites, which install a special handler to kill a test thread instead.

You probably want to use something like `.ok_or_else(|| format_err!("cannot find `token` in OAuth credentials"))?`. It would be fine put that in a helper function so you could write `token_str(&token)?`.

Rust has a bit of a learning curve at first! And `dbcrossbar` does some tricky things with Tokio streams that make it more challenging than usual. But the good news is that there's a finite number of things to learn, so eventually it gets easier.